src/app/home-page/recent-events/recent-events.component.html

```diff
@@ -2,7 +2,7 @@
   <div *ngIf="!(events$ | async) || (events$ | async).length === 0">
     <mat-card class="alert alert-info mx-1" role="alert">
       <mat-icon>info</mat-icon> No events found. Star entries or organizations to see events for them. Read
-      <a [href]="starringDocUrl" target="_blank" rel="noopener noreferrer">{{ starringDocUrl }}</a>
+      <a [href]="starringDocUrl" target="_blank" rel="noopener noreferrer">starring-tools-and-workflows</a>
```

Comment:
Will this display literally "starring-tools-and-workflows"?
Would prefer a more readable "Starring Tools and Workflows" if not already happening